com.realm667.WolfenDoom_Blade_of_Agony.json

```diff
@@ -22,7 +22,7 @@
 		"/share/man"
 	],
 	"modules": [
-        {
```

Comment:
Could you fix the whitespace?

Reply:
That line was using spaces instead of tabs for indentation (the rest of the file uses tabs for indentation), so I made it consistent with the rest. In fact, it even used to be misaligned with a tab size of 4 in the GitHub code viewer (I'm using an userstyle that sets tab width to 4):

![image](https://user-images.githubusercontent.com/180032/42108229-347db2c4-7bda-11e8-8ac6-7f28da46a6dc.png)

Reply:
Switching to tabs is good, what I meant is that the other modules are indented 2 tabs, not 4.